site/content/docs/5.2/components/list-group.md

```diff
@@ -269,6 +269,31 @@ Place Bootstrap's checkboxes and radios within list group items and customize as
 </ul>
 {{< /example >}}
 
+{{< example >}}
+<ul class="list-group">
+  <li class="list-group-item">
+    <input class="form-check-input me-1" type="radio" name="listGroupRadio" value="" aria-label="...">
```

Comment:
In a set of radio buttons, one should be checked by default. You could add `checked` for the first one: `<input class="form-check-input me-1" type="radio" name="listGroupRadio" value="" aria-label="..." checked>`